packages/containers/src/List/List.container.js

```diff
@@ -102,6 +102,13 @@ class List extends React.Component {
 			list: {
 				items,
 				columns: get(this.props, 'list.columns', []),
+				sort: {
+					field: state && state.sortOn,
+					isDescending: state && !state.sortAsc,
```

Comment:
Test if `state` is truthy is useless, it is never falsy.

Reply:
:+1: